### PR TITLE
Add some community docs

### PR DESCRIPTION
**Pull request by adam-palazzo**
@RichardLitt  added a code of conduct and contributing file, and I made a couple of small edits to package.json and the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) fo
 
 * [orbit-db](https://github/com/orbitdb/orbit-db)
 
+```sh
+$ npm install orbit-db-cache
```
Comment (RichardLitt) [inline README.md]: I would add `--save` to this.

File: package.json [hunk 2]
```diff
@@ -2,13 +2,26 @@
...
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "standard --env=mocha"
   },
   "author": "Haad",
+  "contributors": [
+    "RichardLitt",
+    "mistakia",
+    "thiagodelgado111",
+    "adam-palazzo"
```
Comment (RichardLitt) [inline package.json]: I think these ought to be alphabetical? There's a tool in Atom to automatically do that.

File: package.json [hunk 3]
```diff
@@ -2,13 +2,27 @@
...
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "standard --env=mocha"
   },
   "author": "Haad",
+  "contributors": [
+    "adam-palazzo",
+    "mistakia",
+    "RichardLitt",
+    "thiagodelgado111"
+
```
Comment (RichardLitt) [inline package.json]: Extra line?

File: README.md [hunk 4]
```diff
@@ -6,6 +6,10 @@ Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) fo
 
 * [orbit-db](https://github/com/orbitdb/orbit-db)
 
+```sh
+$ npm install --save orbit-db-cache
```
Comment (RichardLitt) [inline README.md]: Just remove `$` for most of these.

File: package.json (hunk 2 above)
Comment (adam-palazzo) [reply package.json]: Alphabetical instead of in order of contribution?

Comment (RichardLitt) [reply package.json]: Yeah. I don't think order of contribution is clear.

File: README.md [hunk 5]
```diff
@@ -1,36 +1,40 @@
-## orbit-db-cache
+# orbit-db-cache
+
+> Local cache for orbit-db
 
 Isomorphic cache used by `orbit-db`, implemented using `level.js` (indexedDB) for the browser and `leveldown` (leveldb) for node.
 
-### Used in
+## install
```
Comment (RichardLitt) [inline README.md]: Capitalize

**Comment by RichardLitt** [comment]
Edited and merged in 593d6cae3936e75b9ab7a6b08daa9a5f715b14ef. Thanks, @adam-palazzo.

**Pull request closed without merging**